content/en/docs/concepts/storage/volume-snapshots.md

```diff
@@ -141,12 +142,40 @@ spec:
   driver: hostpath.csi.k8s.io
   source:
     snapshotHandle: 7bdd0de3-aaeb-11e8-9aae-0242ac110002
+  sourceVolumeMode: Filesystem
   volumeSnapshotRef:
     name: new-snapshot-test
     namespace: default
 ```
 
 `snapshotHandle` is the unique identifier of the volume snapshot created on the storage backend. This field is required for the pre-provisioned snapshots. It specifies the CSI snapshot id on the storage system that this `VolumeSnapshotContent` represents.
+`sourceVolumeMode` is the mode of the volume whose snapshot is taken. It can be either “Filesystem” or “Block”. If not specified, it indicates the source volume's mode is unknown. This field is available in Volume Snapshot client v6.0.0 and higher.
```

Comment:
```suggestion
`sourceVolumeMode` is the mode of the volume whose snapshot is taken. The value of the `sourceVolumeMode` field can be either `Filesystem` or `Block`. If you do not specify a source volume mode, Kubernetes treats the snapshot as if the source volume's mode is unknown. Support for this field is available in VolumeSnapshot client v6.0.0 and higher.
```

If someone asked me for the URL of the VolumeSnapshot client, I would not have a clue what that URL is or how to work it out. We should make it clear what we mean by the phrase “VolumeSnapshot client v6.0.0”.

Reply:
Thanks for the suggestion! I've used most of it but modified `If you do not specify a source volume mode` to `If the source volume mode is not specified`. It is not neccesarily on the user to populate this field.